samples/language/Ternary.qs

```diff
@@ -0,0 +1,15 @@
+/// # Sample
+/// Ternary
+///
+/// # Description
+/// Q# supports conditional expressions (`if` expressions) without the use of a ternary operator.
```

Comment:
```suggestion
/// Q# supports conditional expressions through a ternary operator of the form `cond ? ifTrue | ifFalse`.
```

Reply:
We actually have a ternary operator: https://learn.microsoft.com/en-us/azure/quantum/user-guide/language/expressions/conditionalexpressions